python/dgl/graphbolt/itemset.py

```diff
@@ -36,7 +36,7 @@ def __len__(self):
         raise NotImplementedError
 
 
-class DictItemSet:
+class ItemSetDict:
     r"""Itemset wrapping multiple itemsets with keys.
```

Comment:
Remember to change the docstring accordingly.

Reply:
got it